src/components/search/SuggestionProductCard/suggestion-product-card.scss

```diff
@@ -3,6 +3,10 @@
 .suggestion-product-card {
   padding: var(--fs-spacing-1) 0;
 
+  [data-card-image] {
+    display: flex;
```

Comment:
Fix `[data-card-image]` having more height (62px) than needed (56px) from the image it wraps. 
Is there any other way to do it?

![CleanShot 2022-04-25 at 20 58 54](https://user-images.githubusercontent.com/381395/165193364-3ace753d-0124-4aea-8e43-886ca639c183.gif)

Reply:
Nice!!! I think that is the easiest way to fix it, I'll see if there is another way to do it 💭